terraform/aks/database.tf

```
@@ -15,3 +15,20 @@ module "postgres" {
   server_version          = "14"
   azure_enable_backup_storage  = var.azure_enable_backup_storage
 }
+
+module "redis-cache" {
+  source = "./vendor/modules/aks//aks/redis"
+
+  namespace             = var.namespace
+  environment           = var.environment
+  azure_resource_prefix = var.azure_resource_prefix
+  service_short         = var.service_short
+  config_short          = var.config_short
+  service_name          = var.service_name
+
+  cluster_configuration_map = module.cluster_data.configuration_map
+
+  use_azure               = true
+  azure_enable_monitoring = var.enable_monitoring
+  azure_patch_schedule    = [{ "day_of_week": "Sunday", "start_hour_utc": 01 }]
```

```suggestion
  azure_patch_schedule    = [{ "day_of_week": "Sunday", "start_hour_utc": 01 }]
  azure_maxmemory_policy  = "allkeys-lfu"
```
The schools experience PAAS application redis instance uses allkeys-lfu as its maxmemory policy as shown here https://github.com/DFE-Digital/schools-experience/blob/master/terraform/paas/services.tf line 53 which differs from the terraform-modules default which is allkeys-lru so we need this variable to updated here

---

added